src/main/resources/rp-start

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```

Comment:
The start script that sbt-native-packager uses requires bash, so we already (i.e. prior to this PR) require bash to be part of the base image.

There's room for a future enhancement here to save the 2MB bash adds to the container by writing our own JVM startup script and not relying on this bash syntax: `${!IDX_VAR}`

*note: don't use its AshScript feature, it's broken in various ways*